source/includes/fundamentals/code-examples/LinqAggregation.cs

```diff
@@ -0,0 +1,45 @@
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Bson.Serialization.Conventions;
+using MongoDB.Driver;
+using MongoDB.Driver.Linq;
+
+public class Aggregation
+{
+    // Replace with your connection string
+    private const string MongoConnectionString = "<YOUR_CONNECTION_STRING>";
+
+    public static void Main(string[] args)
+    {   
+        var mongoClient = new MongoClient(MongoConnectionString);
+
+        var database = mongoClient.GetDatabase("sample_restaurants");
+        var collection = database.GetCollection<Restaurant>("restaurants");
+
+        // begin-aggregation
+        // Defines a queryable collection object as a prerequisite to using LINQ
+        var queryableCollection = collection.AsQueryable();
+
+        // Executes the $match and $group aggregation stages
```

Comment:
Just to be precise, this statement defines the query, and the next `ToList` call actually executes it.